toolchain/lower/file_context.cpp

```diff
@@ -233,22 +234,14 @@ auto FileContext::BuildFunctionDecl(SemIR::FunctionId function_id)
 
   // Compute the return type to use for the LLVM function. If the initializing
   // representation doesn't produce a value, set the return type to void.
+  // TODO: Remap return type to i32 if `Run` doesn't return a value.
```

Comment:
```suggestion
  // TODO: For the `Run` entry point, remap return type to i32 if it doesn't return a value.
```

Note, I'm really just trying to capture that `Run` means the entry point. I think that was previously apparent next to the `IsEntryPoint` before, but I think it's not as obvious now.